spec/support/compose_helper.rb

```diff
@@ -0,0 +1,32 @@
+module ComposeHelper
+  def self.app_services
+    @app_services ||= all_services.select do |service_name, _service|
+      service_name =~ /app(-\w+)?$/
+    end
+  end
+
+  def self.rails_services
+    app_services.select do |_service_name, service|
+      service["command"] =~ /rails/
+    end
+  end
+
+  def self.lite_services
```

Comment:
I'd be tempted to move this above `rails_services`, just because we seem to have 'app' and 'lite' stacks for apps, and 'rails services' seem a bit different.

Reply:
Indeed it is - I was wondering if it would confuse people or not. I've renamed to `rails_app_services` to help distinguish it.